kore/src/Kore/Log/DebugAxiomEvaluation.hs

```diff
@@ -123,6 +123,19 @@ data AxiomEvaluationState
 
 instance Entry DebugAxiomEvaluation where
     entrySeverity _ = Debug
+    shortDoc DebugAxiomEvaluation { state } =
+        Just $ Pretty.hcat ["DebugAxiomEvaluation_", showState]
+      where
+        showState =
+            case state of
```

Comment:
This is user-facing text, so it should give more information than this. The `shortDoc` should be about one line long.

Reply:
I modified the instance for a few `State` cases for which I thought it would make sense to use `logWhile`, but please tell me if you had anything else in mind.